apps/api/v1/pages/api/bookings/_get.ts

```diff
@@ -308,7 +299,25 @@ export async function handler(req: NextApiRequest) {
     }
   }
 
-  const data = await prisma.booking.findMany(args);
+  let data: Booking[] = [];
+
+  if (attendeeEmails.length > 0) {
```

Comment:
Instead of keeping this check in the `where` `or` clause, bringing here to run a separate query to get the bookings where the attendee email matches.